### ISSUE TITLE
Allow FederatedStore to be used with Schemas containing conflicting groups

### ISSUE THREAD

**Issue by GCHQDeveloper404** [gchq/Gaffer#1304]
Currently the FederatedStore requires all schemas to be 'mergeable', they must not have conflicting groups.

This issue would be to allow conflicting groups, as long as they are not used together in a single query.

Graph | A | B | C
-|-|-|-
Schema Group | 1 | 1 | 2
property | propX | propY | propZ

Graph A & B would clash, GetAllElements operations should notify the user that this will occur.
Graph A & C is fine.
User1 might not have visibility of Graph B so GetAllElements should work fine with visable Graph A & C.

**Comment by GCHQDeveloper404** [comment]
after #1365

**Comment by GCHQDeveloper404** [comment]
merged into develop
